include/rocksdb/options.h

```diff
@@ -1440,12 +1440,14 @@ struct DBOptions {
   //
   // Time of the day in UTC. Format - HH:mm-HH:mm (00:00-23:59)
   // If the start time > end time, it will be considered that the time period
-  // spans to the next day (e.g., 23:30-04:00)
-  // If the start time == end time, entire 24 hours will be considered offpeak
-  // (e.g. 00:00-00:00). Note that 00:00-23:59 will have one minute gap from
-  // 11:59:00PM to midnight.
+  // spans to the next day (e.g., 23:30-04:00). And start time cannot be equal
+  // to end time. Note that 00:00-23:59 will have one minute gap from 11:59:00PM
+  // to midnight. To set offpeak to be 24hrs, set always_offpeak_override=true.
+  // When always_offpeak_override is true, daily_offpeak_time_utc will be
+  // ignored.
   // Default: Empty String (No notion of peak/offpeak)
   std::string daily_offpeak_time_utc = "";
+  bool always_offpeak_override = false;
 };
```

Comment:
Having two options is not a great solution, as these end up intertwined.  Why not instead have values like "on|always|true" mean that offpeak is always on?

I was fine with start==end meaning offpeak was always ON if it is so documented.  There is already a documented way of offpeak always being off (leaving the value empty).

Reply:
Yeah, I think "always" can be another good option. It'd be even better if 24hr can be covered with valid time range value input. Please take a look at my latest change and let me know what you think.